db/migrate/20210818095622_change_name_column_user.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeNameColumnUser < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :users, :name, :first_name
+    rename_column :users, :surname, :second_name
+    rename_column :orders, :mane_users, :first_name_for_order
```

Comment:
I guess now, until you have production server, better to change old migrations then create new ones!